packages/components/src/components/FileUploader/FileUploader.tsx

```diff
@@ -108,7 +108,7 @@ const FileUploader = React.forwardRef<HTMLDivElement, FileUploaderProps>(
                 {!isMobile && status === "loading" && (
                   <FileUploaderProgressBar progress={progress} />
                 )}
-                {!isMobileUploading && (
+                {!isMobileUploading && (status !== "success" || fileName) && (
```

Comment:
This logic can be hard to understand so my suggestion would be to move it to a variable like 

```javascript
  const isUploadedWithoutName = status === 'success' && !fileName;
```
```suggestion
                {(!isMobileUploading && !isUploadedWithoutName) && (
```

Reply:
Good idea!